.github/workflows/pull-request-podman.yml

```diff
@@ -0,0 +1,69 @@
+name: Podman - run pull-request workflow
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/pull-request-podman.yml'
+      - 'components/container_image.py'
+      - 'directord/components/lib/podman.py'
+      - 'orchestrations/functional-podman-test.yaml'
+
+jobs:
+  functional_podman_functional:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Run install
+        run: sudo bash tools/dev-setup.sh /opt/directord python3 ${{ github.workspace }}
+      - name: Run server service install
+        run: |
+          sudo /opt/directord/bin/directord-server-systemd
+          sudo systemctl daemon-reload
+          sudo systemctl restart directord-server
+      - name: Build current container and run client
+        run: |
+          sudo mkdir /tmp/containers
+          sudo apt-get install -y podman
+          sudo podman build -t test-image .
+          sudo podman run --hostname container-client \
```

Comment:
do we need to run a directord client container here? Feels like we can just run podman directly like so: https://github.com/directord/directord/blob/main/.github/workflows/pull-request.yml#L548-L577

Reply:
I can change it, but this is more fun! :rofl: